api/v1/db/oracledb/game-dao.js

```diff
@@ -0,0 +1,61 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+
```

Comment:
You can remove this extra line, in my previous comment, I meant add a blank line separating `const conn = ...` and `const mergeRawGames = ...`